### PR TITLE
chore(security): upgrade to Alpine 3.15.0 and golang 1.17.6

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by olehermanse** [comment]
@mender-test-bot start pipeline

File: main.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ import (
 )
 
 func main() {
+	// Super malicious comment
```
Comment (mzedel) [inline main.go:38]: It's very concerning that we have to worry about members of our own company rewriting commits - this is once again a quite disappointing behaviour.

**Review by mzedel — CHANGES_REQUESTED** [review]
It's a joy to have a trusted relationship in our company.

**Comment by olehermanse** [comment]
CI is green, this was a test, closing.

**Pull request closed without merging**